### ISSUE TITLE
keras_cv.metrics.BoxCOCOMetrics documentation- Unable to find the parameters and documentation related to coco metrics

### ISSUE THREAD

**Issue by Inshu32** [keras-team/keras-cv#2299]
I am using yolov8 implementation as described on Keras CV page (https://keras.io/examples/vision/yolov8/). 

I wish to print a table with precision, recall and MAP for each class as is done by Yolov7 in Pytorch. 

Something like this: 
![image](https://github.com/keras-team/keras-cv/assets/145352703/1310bee7-2230-4110-886e-19fa671f5a3c)

Is it possible to do this using Keras CV ?

If yes, How ?

**Comment by tirthasheshpatel** [comment]
Yes, you can use PyCOCOMetrics to do this. See this guide: https://keras.io/guides/keras_cv/object_detection_keras_cv

![image](https://github.com/keras-team/keras-cv/assets/43181252/25003215-ffa5-481f-a8f3-6e374cd29138)
